config.go

```diff
@@ -10,6 +10,14 @@ import (
 	"github.com/segmentio/parquet-go/compress"
 )
 
+// ReadMode is an enum that is used to configure the way that a File reads pages.
+type ReadMode int
+
+const (
+	ReadModeAsync ReadMode = iota // ReadModeAsync reads pages asynchronously in the background.
```

Comment:
How about swapping these two constants so the zero value is sync mode?

Reply:
done!